Week3/homework/6-step3.js

```diff
@@ -4,3 +4,22 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 // add your solution here, or add a comment on how you would tackle this problem
+
+// arr2d
+for (let i = 0; i < arr2d.length; i++) {
+  console.log(i, arr2d[i].length);
+  for (let j = 0; j < arr2d[i].length; j++) {
+    console.log(arr2d[i][j]);
+  }
+}
+
+// arr3d
+for (let i = 0; i < arr3d.length; i++) {
+  console.log(i, arr3d[i].length);
+  for (let j = 0; j < arr3d[i].length; j++) {
+    console.log(j, arr3d[i][j].length);
+    for (let n = 0; n < arr3d[i][j].length; n++) {
+      console.log(arr3d[i][j][n]);
+    }
+  }
+}
```

Comment:
The unwanted console.log statements make it difficult to verify which numbers are elements of the array and which numbers are something else.

See the solution I posted in slack how to print an array of any dimension.